tdclient/util.py

```diff
@@ -10,3 +10,27 @@ def create_url(tmpl, **values):
     """
     quoted_values = {k: urlquote(str(v)) for k, v in values.items()}
     return tmpl.format(**quoted_values)
+
+
+def csv_value(s):
```

Comment:
Might be better to name `convert_csv_value` or `validate_csv_value`?

Reply:
I renamed to parse_csv_value.
Thanks.